features/0587-encryption-envelope-v2/README.md

```diff
@@ -0,0 +1,87 @@
+# Aries RFC 0587: Encryption Envelope v2
+
+- Authors: [Baha A. Shaaban](mailto:baha.shaaban@securekey.com) (SecureKey Technologies Inc.), [Troy Ronda](mailto:troy.ronda@securekey.com) (SecureKey Technologies Inc.)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2021-02-10
+- Status Note:
+- Supersedes:
+- Start Date: 2021-02-10
+- Tags: [feature](/tags.md#feature)
+
+## Summary
+
+This RFC proposes that we support the definition of envelopes from [DIDComm Messaging](https://identity.foundation/didcomm-messaging/spec).
+
+## Motivation
+
+This RFC defines ciphersuites for envelopes such that we can achieve better compatability with DIDComm Messaging being specified at DIF.
+The ciphersuites defined in this RFC are a subset of the definitions in [Aries RFC 0334-jwe-envelope](https://github.com/hyperledger/aries-rfcs/tree/master/features/0334-jwe-envelope).
+
+## Encryption Algorithms
+
+[ECDH-1PU draft 03](https://tools.ietf.org/html/draft-madden-jose-ecdh-1pu-03) defines the JWE structure. The following sections summarize the supported algorithms.
+
+### Curves
```

Comment:
DIDComm v2 requires at least X25519 and P-256 and also allows for P-384 and P-521.

Note - It’s worthwhile to also support P-384 and would be preferable for some implementers as compared to P-256.

Reply:
Added clarification text to RFC.

Reply:
Looks good to me based on what we have in the DIDCommV2 spec at the moment.